hack/update-codegen-parallel.sh

```diff
@@ -0,0 +1,248 @@
+    -h "${PROJECT_ROOT}/hack/LICENSE_BOILERPLATE.txt"
+}
+export -f gardenlet_groups
+
+# Componentconfig for admission plugins
+
+shoottolerationrestriction_groups() {
+  bash "${PROJECT_ROOT}"/vendor/k8s.io/code-generator/generate-internal-groups.sh \
+    deepcopy,defaulter \
+    github.com/gardener/gardener/pkg/client/componentconfig \
+    github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis \
+    github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis \
+    "shoottolerationrestriction:v1alpha1" \
+    -h "${PROJECT_ROOT}/hack/LICENSE_BOILERPLATE.txt"
+  
+  bash "${PROJECT_ROOT}"/vendor/k8s.io/code-generator/generate-internal-groups.sh \
+    conversion \
+    github.com/gardener/gardener/pkg/client/componentconfig \
+    github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis \
+    github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis \
+    "shoottolerationrestriction:v1alpha1" \
+    --extra-peer-dirs=github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis/shoottolerationrestriction,github.com/gardener/gardener/plugin/pkg/shoot/tolerationrestriction/apis/shoottolerationrestriction/v1alpha1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/conversion,k8s.io/apimachinery/pkg/runtime,k8s.io/component-base/config,k8s.io/component-base/config/v1alpha1 \
+    -h "${PROJECT_ROOT}/hack/LICENSE_BOILERPLATE.txt"
+}
+export -f shoottolerationrestriction_groups
+
+# OpenAPI definitions
+
+openapi_definitions() {
+  echo "Generating openapi definitions"
+  rm -Rf ./${PROJECT_ROOT}/openapi/openapi_generated.go
+  go install ./${PROJECT_ROOT}/vendor/k8s.io/kube-openapi/cmd/openapi-gen
+  ${GOPATH}/bin/openapi-gen "$@" \
+    --v 1 \
+    --logtostderr \
+    --input-dirs=github.com/gardener/gardener/pkg/apis/core/v1alpha1 \
+    --input-dirs=github.com/gardener/gardener/pkg/apis/core/v1beta1 \
+    --input-dirs=github.com/gardener/gardener/pkg/apis/settings/v1alpha1 \
+    --input-dirs=github.com/gardener/gardener/pkg/apis/seedmanagement/v1alpha1 \
+    --input-dirs=k8s.io/api/core/v1 \
+    --input-dirs=k8s.io/api/rbac/v1 \
+    --input-dirs=k8s.io/api/autoscaling/v1 \
+    --input-dirs=k8s.io/apimachinery/pkg/apis/meta/v1 \
+    --input-dirs=k8s.io/apimachinery/pkg/api/resource \
+    --input-dirs=k8s.io/apimachinery/pkg/types \
+    --input-dirs=k8s.io/apimachinery/pkg/version \
+    --input-dirs=k8s.io/apimachinery/pkg/runtime \
+    --input-dirs=k8s.io/apimachinery/pkg/util/intstr \
+    --report-filename=${PROJECT_ROOT}/pkg/openapi/api_violations.report \
+    --output-package=github.com/gardener/gardener/pkg/openapi \
+    -h "${PROJECT_ROOT}/hack/LICENSE_BOILERPLATE.txt"
+}
+export -f openapi_definitions
+
+parallel ::: \
```

Comment:
On my machine, this seems to do nothing at all, while substituting `:::` with `--` works perfectly fine. 🤔

Reply:
Did you run `brew install parallel`?

Reply:
Yes, but I already had `parallel` installed as part of the `moreutils` package

Reply:
Perhaps you have a different version of `parallel`? Though according to the documentation, `:::` is what should work, not `--`.

My version is:
```
$ parallel --version
GNU parallel 20201122
...
```

Reply:
Ok, just confirmed that those are two different versions, that are not compatible with regards to argument handling...
When unlinking `moreutils` and linking `parallel` instead, it works...
This cries out for https://github.com/gardener/gardener/issues/2143